README.md

```diff
@@ -262,10 +257,13 @@ that you wish the world to use:
 
 ### Adjusting global server properties
 Default values in the script can be overridden by adding certain properties to one
-of the `mscs.defaults` files. The `mscs.defaults` files can be found found in one
+of the `mscs.defaults` files. The `mscs.defaults` files needs to be created in one
 of three places depending on how the script is being used. When using the `mscs`
-script, the `mscs.defaults` file can be found at `/opt/mscs/mscs.defaults`. When
-using the `msctl` script in [multi-user mode](https://github.com/MinecraftServerControl/mscs/wiki/Configuring-MSCS-for-multiple-users), the `mscs.defaults` file can be found at either `$HOME/mscs.defaults` or `$HOME/.config/mscs/mscs.defaults`.
+script, the `mscs.defaults` file __**will need to be created*__ at `/opt/mscs/mscs.defaults`. When
```

Comment:
you have 2 asterisks before, one after.  this leaves 1 asterisk in the text when I don't think you meant to.